requirements/tools.in

```diff
@@ -12,3 +12,4 @@ twine
 attrs
 coverage
 pyupio
+django
```

Comment:
Adding django to the tools venv is fine (the specific thing you wanted me to comment on).